src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -0,0 +1,499 @@
+mv peer.crt cert
+mv peer.key key
+mv ca.crt cacert
+
+kubectl -n newrelic create secret generic newrelic-etcd-tls-secret --from-file=./cert --from-file=./key --from-file=./cacert
+```
+
+Finally, you can input the secret name (`newrelic-etcd-tls-secret`) and namespace (`newrelic`) in the config snippet shown at the beginning of this section. Remember that the Helm Chart will automatically parse this config and create an RBAC role to grant access to this specific secret and namespace for the `nrk8s-controlplane` component, so there's no manual action needed in that regard.
+
+### Static endpoints [#static-endpoints]
+
+While autodiscovery should cover cases where the control plane lives inside the Kubernetes clusters, some distributions or sophisticated Kubernetes environments run the control plane elsewhere, for a variety of reasons including availability or resource isolation.
+
+For these cases, the integration can be configured to scrape an arbitrary, fixed URL regardless of whether a pod with a control plane label is found in the node. This is done by specifying a `staticEndpoint` entry. For example, one for an external etcd instance would look like this:
+
+```yaml
+controlPlane:
+  etcd:
+    staticEndpoint:
+      url: https://url:port
+      insecureSkipVerify: true
+      auth: {}
+```
+
+`staticEndpoint` is the same type of entry as `endpoints` in the `autodiscover` entry, whose fields are described above. The authentication mechanisms and schemas are supported here.
+
+Please keep in mind that if `staticEndpoint` is set, the `autodiscover` section will be ignored in its entirety.
+
+#### Limitations [#static-endpoints-limitations]
+
+<Callout variant="important">
+  If you are using `staticEndpoint`, you must change `controlPlane.kind` from `DaemonSet` to `Deployment`.
+</Callout>
+
+When using `staticEndpoint`, all `nrk8s-controlplane` pods will attempt to reach and scrape said endpoint. This means that, if `nrk8s-controlplane` is a DaemonSet (the default), your components will get scraped multiple times, potentially leading to duplicate metrics and increased billable usage. If you are using `staticEndpoint`, make sure to change `controlPlane.kind` to Deployment.
+
+For the same reason above, it is currently not possible to use autodiscovery for some control plane components, and a static endpoint for others. This is a know limitation we are working to address in future versions of the integration.
+
+Lastly, `staticEndpoint` allows only to define a single endpoint per component. This means that if you have multiple control plane shards in different hosts, it is currently not possible to point to them separately. This is also a known limitation we are working to address in future versions. For the time being, a workaround could be to aggregate metrics for different shards elsewhere, and point the `staticEndpoint` URL to the aggregated output.
+
+## Monitoring control plane with integration version 2 [#monitoring-control-plane]
```

Comment:
Should we keep at least during the beta phase this section in the upper part?

Reply:
I'm not sure it will impact most users, I'm worried it could be too noisy if we pin it at the top.

Reply:
resolved